src/DotNetty.Common/Concurrency/TaskCompletionSource.cs

```diff
@@ -34,6 +34,11 @@ public bool setUncancellable()
             return true;
         }
 
+        public override string ToString()
+        {
+            return "TaskCompletionSource[status: " + this.Task.Status.ToString() + "]";
```

Comment:
+ .ToString() is redundant, right? It should auto-convert based on inferring the '+' op